src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+
+### Consistency Within a Trace
+
+For traces that involve multiple transactions, Sentry uses a "head-based" approach: a sampling decision is made in the originating service, and then that decision is passed to all subsequent services. To see how this works, let's return to our webapp example above. Consider two users, A and B, who are both loading your app in their respective browsers. When A loads the app, the SDK pseudorandomly "decides" to collect a trace, whereas when B loads the app, the SDK "decides" not to. When each browser makes requests to your backend, it includes in those requests the "yes, please collect transactions" or the "no, not collecting transactions this time" decision in the headers. 
+
+When your backend processes the requests from A's browser, it sees the "yes" decision, collects transaction and span data, and sends it to Sentry. Further, it includes the "yes" decision in any requests it makes to subsequent services (like your database server), which similarly collect data, send it to Sentry, and pass the decision along to any services they call. Through this process, all of the relevant transactions in A's trace are collected and sent to Sentry.
+
+On the other hand, when your backend processes the requests from B's browser, it sees the "no" decision, and as a result it does _not_ collect and send transaction and span data to Sentry. It does, however, do the same thing it does in A's case in terms of propagating the decision to subsequent services, telling them not to collect or send data either. They then in turn tell any services they call not to send data, and in this way no transactions from B's trace are collected.
+
+Put simply: as a result of this head-based approach, where the decision is made once in the originating service and passed to all subsequent services, either all of the transactions for a given trace are collected, or none are, so there shouldn't be any incomplete traces.
+
+### Consistency Between Traces
+
+If you enable tracing in services with multiple entry points, we recommend choosing similar sampling rates, to avoid biasing your data. For example, suppose the backend of our on-going webapp example also serves as a public API. In that case, some traces would start with a pageload transaction in the web app, and likely include multiple backend transactions, while other traces (those representing folks hitting the public API) would begin with a single backend request transaction, which would be the only backend transaction in the trace. Choosing a very different sampling rate for your web app from that chosen for your backend would lead to one of those scenarios being oversampled compared to the other, compromising the accuracy of your overall data.
```

Comment:
Yeah - that all makes a ton of sense. This is an example of me rewriting and fleshing out content which previously existed. Here how the idea is expressed in the current docs:

` If your services have multiple entry points, you should aim to choose a consistent sampling rate for each. Choosing different sampling rates can bias your results.`

Perhaps @dashed or @markstory can address this question?

Reply:
That makes sense to me. 

@rhcarvalho brings up a good point that data should be analyzed in isolation if sampling rates at various sources differ.

Reply:
So what's the consensus here? Should I

1. Take out this part entirely, since it might not be great advice,

or

2. Modify it so that the focus is not consistent rates but on data bias in general when you have multiple entry points, and mention Rodolfo's tags idea

?

Reply:
@rhcarvalho do you something specific that can be changed? Otherwise, we can keep this for now and address it later once we have a more concrete opinion on this.

Reply:
> Maybe the use of `tags` could be helpful to filter each data set?

We only filter on transaction name for now, so the `tags` advice is not immediately useful.

I don't want to slow this PR down, so we can track the discussion in #1678.